src/test/java/eu/europeana/metis/sandbox/SandboxApplicationTests.java

```diff
@@ -6,8 +6,8 @@
 @SpringBootTest
 class SandboxApplicationTests {
 
-	@Test
-	void contextLoads() {
-	}
+  @Test
+  void contextLoads() {
+  }
```

Comment:
Is this test supposed to be empty?

Reply:
This one is added by the initialzr when creating the project, its purpose its to be an Integration test to validate that the application context loads successfully, which in practice is more useful than it sounds

Reply:
As long as it does something useful :)
Perhaps you could include a comment to that effect in the code?

Reply:
Sure I will do that now